crates/examples/Cargo.toml

```diff
@@ -65,6 +65,10 @@ path = "push-cdn/all.rs"
 name = "validator-push-cdn"
 path = "push-cdn/validator.rs"
 
+[[example]]
+name = "multi-validator-push-cdn"
+path = "push-cdn/multi-validator.rs"
+
```

Comment:
Just curious - are you using this one for local benchmarking?

Reply:
Oh yes, I'm using it for local benchmarking, and when running validators on EC2 failed I'll use this for debugging.